lib/integration/queryManager.js

```diff
@@ -0,0 +1,34 @@
+import { logMessage } from "../logger";
+import { dbConnector } from "./dbConnector";
+
+export const executeQuery = async (sql, params) => {
+  const client = await dbConnector();
+  return client
+    .query(sql, params)
+    .then((data) => buildResult(data))
```

Comment:
Don't think this is a good idea. This function should just execute the query and handle any errors. buildResult should be called separately